storage/innobase/xtrabackup/test/bootstrap.sh

```diff
@@ -31,6 +31,20 @@ function ssl_version() {
     echo ${sslv}
 }
 
+function glibc_version() {
+    glibc=$(ldd --version | head -1 | awk '{print $NF}')
+    case ${glibc} in
+        2.12|2.17|2.27|2.28|2.31|2.34|2.35) ;;
+        *)
+            if ! test -r "${1}"; then
```

Comment:
where do we get ${1} from?

Reply:
I think this all originated from https://github.com/percona/percona-xtrabackup/commit/6d0b9ea4172cb4b4c738838591444c2206dd858e , and I think this commit got the `if ! test -r "${1}"; then` from the main loop. In the SSL and GLIBC context this is not required. I will remove